src/components/node-list/styles/_group.scss

```diff
@@ -46,22 +42,44 @@ $placeholder-fade: 120px;
   pointer-events: none;
 }
 
-.pipeline-nodelist__placeholder-upper:after {
-  bottom: -$placeholder-fade;
-  background: linear-gradient(
-    0deg,
-    var(--nodelist-bg-transparent) 0%,
-    var(--color-bg-3) 100%
-  );
+.pipeline-nodelist__elements-panel {
+  .pipeline-nodelist__placeholder-upper:after {
+    bottom: -$placeholder-fade;
+    background: linear-gradient(
+      0deg,
+      var(--nodelist-bg-transparent) 0%,
+      var(--color-nodelist-element-panel) 100%
+    );
+  }
+
+  .pipeline-nodelist__placeholder-lower:after {
+    top: -$placeholder-fade;
+    background: linear-gradient(
+      0deg,
+      var(--color-bg-3) 0%,
+      var(--color-nodelist-element-panel) 100%
+    );
+  }
 }
 
-.pipeline-nodelist__placeholder-lower:after {
-  top: -$placeholder-fade;
-  background: linear-gradient(
-    0deg,
-    var(--color-bg-3) 0%,
-    var(--nodelist-bg-transparent) 100%
-  );
+.pipeline-nodelist__filter-panel {
+  .pipeline-nodelist__placeholder-upper:after {
```

Comment:
is this a class that is supposed to be deleted and replaced once the tree items are merged in?

Reply:
Check the latest changes, but this is still around as the filters panel uses this as part of the lazy list loading effect.